code/game/objects/items.dm

```diff
@@ -196,6 +196,8 @@
 		if(src == user.get_inactive_hand())
```

Comment:
Looks like the `put_in_hand_check()` call works just fine without checking whether the item is in the user's inactive hand anyway.